### DISCUSSION TITLE
How to recreate the kbar from the docs?

### DISCUSSION THREAD

**Discussion by parotech123** [skeletonlabs/skeleton#1102] [Q&A]
### Describe what feature you'd like. Pseudo-code, mockups, or screenshots of similar solutions are encouraged!

I was looking to recreate the beautiful kbar from the docs.

Any help on how to proceed?

Thanks!

### What type of pull request would this be?

None

**Comment by endigo9740** [comment]
@parotech123 we don't distribute or really provide support for our internal documentation components. Though we sometimes "dogfood" these internally before packaging them into the library.

One thing to note though, I'd recommend looking at the latest iteration as part of #1080. I've redesign it to be much more visually appealing. The code is a bit cleaner as well:

![Screenshot 2023-03-06 at 5 25 23 PM](https://user-images.githubusercontent.com/1509726/223281575-3e2ddaf5-e0d0-4f40-9972-a3491381f351.png)

View the source here:
https://github.com/skeletonlabs/skeleton/blob/8edb8c989984a3e208dfac18448dd6d214972148/src/docs/modals/DocsSearch/DocsSearch.svelte

General idea is it loops through are navigation tree, which is stored in a static .ts file, and then we iterate and filter the results of those:
https://github.com/skeletonlabs/skeleton/blob/dev/src/docs/links.ts

**Comment by endigo9740** [comment]
For now this is not an "actionable" item, so I'm going to bump this over to a Discussion instead.
